CONTRIBUTING.md

```diff
@@ -16,128 +16,158 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
-PRs must follow the following naming conventions:
+### Adding a new ________?
 
-* naming must be logical
-* under `notebooks` use the naming convention *scenario-framework-etc-compute*, where *scenario* is one of ["train", "deploy", "score", "dprep"]
-* directories under `tutorials` must be words separated by hyphens
-* tutorial workflows (and workflow files) use the naming convention `run-tutorial-*initials*`, where *initials* is the initials of the words
+Thinking of contributing a new example? Read this first!
 
-### Testing
+#### Tutorials
 
-PRs must include necessary changes to any testing to ensure:
+A tutorial is an end-to-end example accomplishing something significant or teaching how to scale up and out in the cloud. A tutorial **must** have an excellent `README.md` file in its directory, following conventional markdown syntax, explaining:
 
-* `run-examples` runs on every push and PR to `main` (with changes to examples) and runs all examples under `examples/`
-* `run-notebooks` runs on every push and PR to `main` (with changes to notebooks) and runs all examples under `notebooks/`
-* `run-tutorial-initials` must be tested at least daily and on PR to `main` (with changes to the tutorial)
-* `cleanup` runs daily and cleans up AML resources for the testing workspace
-* `smoke` runs hourly and on every push and PR to `main` and performs sanity checks
+- required prerequisites
+- any one-time setup needed by the user
+- any other setup instructions
+- overview of files in the tutorial
+- relevant links
 
-### Miscellaneous
+Tutorials are often, but not required to be, a series of ordered Jupyter notebooks. All Jupyter notebooks must utilize notebook features (i.e. be interactive, have explanation in markdown cells, etc).
 
-* to modify `README.md`, you need to modify `readme.py` and accompanying markdown files
-* the tables in the `README.md` are auto-generated, including description, via other files
-* develop on a branch, not a fork, for workflows to run properly
-* use an existing environment where possible
-* use an existing dataset where possible
-* don't register environments
-* don't create compute targets
-* don't modify `requirements.txt`
-* you probably shouldn't modify any files in the root of the repo
-* you can `!pip install --upgrade packages` as needed in notebooks
+**You should probably ask (open an issue) before contributing a new tutorial.** Currently, themes for tutorials include:
 
-### Unenforced naming
+- `using-*` for learning ML tooling basics and tracking/scaling in the cloud
+- `work-with-*` for integrations with cloud tooling, e.g. `work-with-databricks`, `work-with-synapse`
+- `deploy-*` for advanced deployment scenarios
+- `automl-with-*` for automated ML tutorials
+
+#### Notebooks
+
+A notebook is an example accomplishing something significant in a Jupyter notebook, often written in Python. To qualify to be a notebook, the example must:
+
+- obviously benefit from being a Jupyter notebook
```

Comment:
What makes something _not_ obviously benefit from being a Jupyter notebook?

I.e., do you allow "raw markdown / Python" examples?